prov/efa/src/rdm/efa_rdm_peer.h

```diff
@@ -151,7 +151,7 @@ bool efa_rdm_peer_support_delivery_complete(struct efa_rdm_peer *peer)
 static inline
 bool efa_both_support_rdma_read(struct rxr_ep *ep, struct efa_rdm_peer *peer)
 {
-	return efa_domain_support_rdma_read(rxr_ep_domain(ep)) &&
+	return efa_endpoint_support_rdma_read(ep) &&
```

Comment:
the function name should be `efa_rdm_ep_support_rdma_read`

Reply:
The functions are now named efa_rdm_ep_support_rdma_read and *_rdma_write.